CS342MemberViews/Sources/SonyaJin/ProfileView.swift

```diff
@@ -0,0 +1,37 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+import SwiftUI
+
+
+public struct ProfileView: View {
+    public var body: some View {
+        profileImage
+            .resizable()
+            .aspectRatio(1.0, contentMode: .fit)
```

Comment:
You can add the `scaledToFill()` modifier to fix the squeezing of your image:
```suggestion
            .resizable()
            .scaledToFill()
            .aspectRatio(1.0, contentMode: .fit)
```

Reply:
@sonyashijin I have committed a fix to your issue here to get you started with adding some more creative views.